### ISSUE TITLE
opam should display a warning for files not listed in extra-files

### ISSUE THREAD

**Issue by dra27** [ocaml/opam#3418]
Checksums are verified if files are listed, but I think that unlisted files are silently used?

**Comment by AltGr** [comment]
Yes. The idea is to have this field for internal validation (ie detecting file changes for triggering recompilations), and in time, to enforce the field when we introduce end-to-end package signing.

**Comment by rjbou** [comment]
A warning is displayed, in debug mode, cf. #4198
